### ISSUE TITLE
Custom template not rendering on Google Cloud Run

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-auth-github#32]
https://datasette-auth-demo-j7hipcg4aq-uc.a.run.app/

<img width="853" alt="Datasette__fixtures_and_fixtures__roadside_attractions__4_rows" src="https://user-images.githubusercontent.com/9599/60778636-9ab83400-a0ec-11e9-9702-6f962393140c.png">

No logout link, which is odd because https://datasette-auth-demo-j7hipcg4aq-uc.a.run.app/-/plugins says:

```json
[
    {
        "name": "datasette_auth_github",
        "static": false,
        "templates": true,
        "version": "0.6.1"
    }
]
```

**Comment by simonw** [comment]
Deployed using:
```
datasette publish cloudrun fixtures.db \
  --branch=master \
  --install=datasette-auth-github \
  -n datasette-auth-demo \
  --service datasette-auth-demo \
  --show-files \
  --plugin-secret datasette-auth-github client_id 7524167294949f7886fc \
  --plugin-secret datasette-auth-github client_secret ...
```

**Comment by simonw** [comment]
I'll try the above command again once I've released Datasette 0.29 (but without the `--branch=master` bit).

**Comment by simonw** [comment]
This is really weird. It seems to only be a problem on Cloud Run.

Here's Heroku: https://datasette-auth-demo.herokuapp.com/

Deployed using
```
datasette \
  publish heroku fixtures.db \
    --name datasette-auth-demo \
    --install=datasette-auth-github \
    --plugin-secret datasette-auth-github client_id 808bcabcea485059283c \
    --plugin-secret datasette-auth-github client_secret ...
```

And here's Zeit Now: https://datasette-auth-demo.now.sh/

Deployed using:
```
datasette \
  publish nowv1 fixtures.db \
    --alias datasette-auth-demo \
    --install=datasette-auth-github \
    --plugin-secret datasette-auth-github client_id 86e397f7fd7a54d26a3a \
    --plugin-secret datasette-auth-github client_secret ...
```
Both of those are showing the custom navigation bar just fine.

Cloud Run: https://datasette-auth-demo-j7hipcg4aq-uc.a.run.app/
```
datasette publish cloudrun fixtures.db \
   --install=datasette-auth-github \
   -n datasette-auth-demo \
   --service datasette-auth-demo \
   --show-files \
   --plugin-secret datasette-auth-github client_id 7524167294949f7886fc \
   --plugin-secret datasette-auth-github client_secret ...
```
It's broken. I don't understand why.

**Comment by simonw** [comment]
OK, it works now! https://datasette-auth-demo-j7hipcg4aq-uc.a.run.app/

I added `--install=datasette-cluster-map` to the end of the deploy command (to test if static files were serving correctly) and that seemed to cause the installation of a more up-to-date package which fixed the bug.

**Comment by simonw** [comment]
Saw this issue again, this time on Glitch.

**Comment by simonw** [comment]
... and it turns out that's because I had installed the plugin on Glitch using this is `requirements.txt`:

`https://github.com/simonw/datasette-auth-github/archive/target-top.zip`

When I switched that back to `datasette-auth-github~=0.8` it started working again.

So for some reason custom templates don't get loaded if you install the plugin from a URL to a zip file as opposed to installing directly from PyPI.

Now that I know the cause I'm closing this again. I wonder if there is a fix for this though? Being able to install from a URL is really useful.
